pages/esmodule.tsx

```diff
@@ -154,7 +134,7 @@ const ESModule = () => {
           )}
         </DialogContent>
       </DialogOverlay>
-      {cageTime && cageTime.gt(0) && (
+      {cageTime && (
```

Comment:
Why remove `cageTime.gt(0)`?

Reply:
It seems that cageTime is a Date object, i think this was a bug. 
https://github.com/makerdao/dai.js/blob/0e679826a841c654ee9c49c81101e6092b486753/packages/dai-plugin-mcd/src/schemas/end.js#L15

It also seems that the format date used below receives a Date object. I think someone has mistaken it by a CurrencyObject or a bignumber. 

But i'm not sure, i don't know how to activate the cageTime